torchci/lib/bot/autoLabelBot.ts

```diff
@@ -388,16 +389,12 @@ function myBot(app: Probot): void {
       const owner = context.payload.repository.owner.login;
       const repo = context.payload.repository.name;
       const title = context.payload.pull_request.title;
-      const filesChangedRes = await context.octokit.paginate(
-        "GET /repos/{owner}/{repo}/pulls/{pull_number}/files",
-        {
-          owner,
-          repo,
-          pull_number: context.payload.pull_request.number,
-          per_page: 100,
-        }
+      const filesChanged = await getFilesChangedByPr(
+        context.octokit,
+        owner,
+        repo,
+        context.payload.pull_request.number
```

Comment:
nit: owner, repo, and pull request number all originally come from context.  Depending on where else you want to use this function, you could just take in context

Reply:
true, but I kinda prefer this more explicit listing of dependencies since not all contexts contain a PR. This way the dependencies are all explicit